docs/api/covidcast-signals/nssp.md

```diff
@@ -0,0 +1,104 @@
+
+This weighting approach assumes that the number of ED visits is proportional to the overall population of a county, i.e. the per-capita ED visit rate is the same for all counties, which may not be the case (for example, denser counties may have easier access to EDs and thus higher rates of ED visits per capita).
+
+State reporting process is separate from the county reporting process. As such, state-level data is **not** simply an average of the county-level data, but may contain facilities omitted at the regional level. For example, state-level values are available for California, even though no California county data is reported.
+
+### Smoothing
+
+Smoothed signals are calculated using a simple 3 week moving average of the relevant weekly signals. Note that since the unsmoothed `pct_visits_*` signals report weekly data, each smoothed signal value is computed from 3 points rather than 21, as would be used if the unsmoothed data were reported daily.
+
+
+## Missingness
+
+As of May 2024, NSSP received data from 78% of US EDs.
+
+[The NSSP site participation map](https://www.cdc.gov/nssp/media/images/2024/04/Participation-with-date.png) shows counties containing at least one reporting ED between January 2023 and April 2024.
+California, Colorado, Missouri, Oklahoma, and Virginia have the most noticeable gaps in coverage, with many component counties having either no eligible EDs or having no recently reported data in NSSP. However, most states have some counties that are absent.
+
+NSSP does not report county-level data for all counties with reporting EDs; some reporting EDs are only included in state-level values.
+
+The following states report no data through NSSP at the county level: CA, WA, AK, AZ, AL, CO, SD, ND, MO, AR, FL, OH, NH, CT, NJ.
+
+South Dakota, Missouri, and territories report no data through NSSP at the state level.
+
+
+## Lag and Backfill
+
+The weekly signal is reported on Friday mornings, adding data from the prior week.
+For example, on Friday, 2024-04-19, the source added new data from the week ending 2024-04-13.
+
+This data source has frequent backfill, primarily arising from newly included EDs. When a new facility joins the reporting network, its historical data is added to the dataset, resulting in changes to historical values for every geographic level that ED is part of (county through nation). Because of this, the broadest geographic levels are more likely to be revised.
+
+In previous revisions, we have noted changes to values dating back about 2 years.
+
+
+## Limitations
+
+There is substantial missingness at the county level. This tends to impact more rural and lower population locations. See the [missingness section](#missingness) for more information.
+
+Not all counties contain reporting EDs, including in states where NSSP reports state-level data.
```

Comment:
This is a stub. I was trying to get at the idea that state-level data is based on a subset of counties/EDs and is probably doesn't have much data from more rural places, but then I saw that Orange county doesn't any reporting EDs and it's definitely not rural. So maybe I'm just trying to say "may be inaccurate but we don't know how"?

Reply:
To be clear, going off [this map](https://www.cdc.gov/nssp/media/images/2024/04/Participation-with-date.png), Orange county could have reporting EDs but just doesn't?

Reply:
Yeah, that map. I was trying to think about patterns in counties that don't have reporting EDs and how that would impact aggregated values. Initially I assumed it would be rural counties (low population, no big hospitals, etc). But some high population counties, like Orange county (3.5 million pop), are also missing.

Apparently ~700 counties don't have hospitals at all.

Anyway, you should take this paragraph where you want/where you think would be useful.